mac/Keyman4MacIM/Keyman4MacIM/KMInputMethodEventHandler.m

```diff
@@ -161,17 +161,18 @@ - (void)updateContextBuffer:(id)sender {
     }
     
     NSRange selRange = [sender selectedRange];
-    NSUInteger len = [sender length];
+    NSUInteger len;//[sender length];
```

Comment:
Let's not leave commented code in; also can you explain the change here?

Reply:
I left that as a reminder to run through the entire set of acceptance tests (which I'm attempting to do) to make sure it's right. In the process of writing the unit test, when I tried to mock out the length property of the sender, I discovered that it's not actually part of the defined protocol. So that means that if there are any clients that implement it (which I'm pretty sure there are), I can't actually count on it to be meaningful (which I'm pretty sure I had noticed before, in at least some cases). So if removing it doesn't seem to break anything, I think it's pretty safe to conclude that it wasn't needed in the first place.

Reply:
Great, that makes sense to me now. Might be good to add a comment accordingly.